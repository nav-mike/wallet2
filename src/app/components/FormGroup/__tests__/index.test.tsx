import * as React from 'react';
import { render } from '@testing-library/react';
import { FromGroup } from '..';

describe('<FormGroup />', () => {
  it('should be render and match the snapshot', () => {
    const group = render(<FromGroup />);
    expect(group.container.firstChild).toMatchSnapshot();
  });
});
