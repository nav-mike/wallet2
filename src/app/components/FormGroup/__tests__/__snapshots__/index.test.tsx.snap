// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormGroup /> should be render and match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 1.3rem;
}

<div
  class="c0"
/>
`;
